src/ports/postgres/modules/graph/pagerank.sql_in

```diff
@@ -120,6 +121,10 @@ distribution per group. When this value is NULL, no grouping is used and
 a single model is generated for all data.
 @note Expressions are not currently supported for 'grouping_cols'.</dd>
 
+<dt> nodes_of_interest (optional) </dt>
```

Comment:
@fmcquillan99 Do we also need additional explanation of personalized pagerank somewhere in the user doc?

Reply:
Yes user doc will need more explanation of what is personalized page rank and how it works and examples.

Reply:
I will add more explanation to the docs on how personalized page rank works. And still working on Test cases.